### ISSUE TITLE
Info about fridge and saddle

### ISSUE THREAD

**Issue by piotrch** [cadon/ARKStatsExtractor#1067]
We could possibility of adding a short line of description in which cryo fridge creature is stored and also mark if it is frozen with saddle (possibly with colour or even armor %)

**Comment by cadon** [comment]
There's a `Note` entry for each creature, maybe that can be used for that?
Or do you mean that this info somehow should be imported? I'm not sure if the needed info is available.

**Comment by piotrch** [comment]
Yeah note field is some solution I didn't think about. But who like to type note each time - we could have additional field similar to "Server" where you can type some text or choose from dropdown one of texts entered before. It could look like:
![image](https://user-images.githubusercontent.com/4151233/79788858-95f60100-8349-11ea-8430-f81cf3c92214.png)
